api/shape/sort.js

```diff
@@ -7,65 +7,65 @@ const X = 0;
 const Y = 1;
 const Z = 2;
 
-export const sort = Shape.registerMethod(
+export const sort = Shape.registerMethod2(
   'sort',
-  (spec = 'z<y<x<') =>
-    async (shape) => {
-      let leafs = [];
-      for (const leaf of getLeafs(await shape.toGeometry())) {
-        const [min, max] = measureBoundingBox(leaf);
-        const shape = await Shape.fromGeometry(leaf);
-        leafs.push({ min, max, shape });
+  ['inputGeometry', 'string'],
```

Comment:
Are you converting all of these by hand or is there some sort of macro to do it?

Reply:
Manually, for my sins. :)

Hopefully it will get simpler over time.

Reply:
😬 Good luck!